spec/support/matchers/graph_matchers.rb

```diff
@@ -21,3 +21,39 @@
     'compares triples, ignoring graph names'
   end
 end
+
+RSpec::Matchers.define :have_quad do |quad|
```

Comment:
Is RSpec doing some magic on lines 25 and 43 to enable usage of both `has` and `have`? I ask because I see you named one matcher with `have` and the other with `has`.

Reply:
My English is not very code. Making consistent (though I could add an alias).

Reply:
Looks like RSpec is doing magic.